### PR TITLE
fix(editor): Fix flaky mapping tests

### PR DESCRIPTION
**Pull request by elsmr**
## Summary

Mapping tests were flaky because of autocomplete onboarding

- Disable autocomplete onboarding in tests
- Test stability improvements
- Remove cy.wait

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

**Review by MiloradFilipovic — DISMISSED** [review]
Looking good but looks like an unrelated test is failing now :/

**Review by ivov — DISMISSED** [review]
🙏🏻

File: cypress/e2e/33-settings-personal.cy.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ describe('Personal Settings', () => {
 			cy.getByTestId('personal-data-form').find('input[name="firstName"]').clear().type(name);
 			cy.getByTestId('personal-data-form').find('input[name="lastName"]').clear().type(name);
 			cy.getByTestId('save-settings-button').click();
-			errorToast().should('contain', 'Malicious firstName | Malicious lastName');
+			errorToast().should('contain', 'Potentially malicious string | Potentially malicious string');
```
Comment (ShireenMissi) [inline cypress/e2e/33-settings-personal.cy.ts:44]: thank you 🙏

Comment (mutdmour) [reply cypress/e2e/33-settings-personal.cy.ts:44]: @elsmr I disabled this test on master. Can you please pull and revert that? thanks

Comment (elsmr) [reply cypress/e2e/33-settings-personal.cy.ts:44]: OK!

**Review by ShireenMissi — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.55.3`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.55.3)

**Pull request merged**